### PR TITLE
IGNITE-11958 JDBC connection validation should use it's own task instead of cache validation task

### PR DESCRIPTION
**Pull request by denis-chudov**

### CODE REVIEW

File: modules/core/src/main/java/org/apache/ignite/internal/jdbc/JdbcConnectionValidationTask.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import org.apache.ignite.IgniteException;
+import org.apache.ignite.compute.ComputeJob;
+import org.apache.ignite.compute.ComputeJobAdapter;
+import org.apache.ignite.compute.ComputeJobResult;
+import org.apache.ignite.compute.ComputeTaskSplitAdapter;
+import org.apache.ignite.internal.util.typedef.F;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * This task is used for JDBCConnection validation.
```
Comment (vmalin) [inline modules/core/src/main/java/org/apache/ignite/internal/jdbc/JdbcConnectionValidationTask.java]: I think it's better to use {@link JCDCConnection}. Especially when we have to version of this class and one is deprecated.

File: modules/core/src/main/java/org/apache/ignite/internal/jdbc/JdbcConnectionValidationTask.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+ *
+ * @deprecated Using Ignite client node based JDBC driver is preferable.
```
Comment (vmalin) [inline modules/core/src/main/java/org/apache/ignite/internal/jdbc/JdbcConnectionValidationTask.java]: Looks like copy-paste from JDBCConnection. Should it be here?

Comment (denis-chudov) [reply modules/core/src/main/java/org/apache/ignite/internal/jdbc/JdbcConnectionValidationTask.java]: I think yes, as JdbcConnection and related things are deprecated and will be removed in ignite 3.

File: modules/core/src/main/java/org/apache/ignite/internal/jdbc/JdbcConnectionValidationTask.java (hunk 1 above)
Comment (denis-chudov) [reply modules/core/src/main/java/org/apache/ignite/internal/jdbc/JdbcConnectionValidationTask.java]: fixed

**Pull request closed without merging**